hackathon_site/hardware/test_api.py

```diff
@@ -864,6 +864,89 @@ def test_successful_post(self):
             self.assertEqual(final_response[attribute], self.request_data[attribute])
 
 
+class IncidentDetailViewGetTestCase(SetupUserMixin, APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.profile = Profile.objects.create(user=self.user)
+        self.team = Team.objects.create()
```

Comment:
Just a side note, your database setup here is fine but you have some objects that are unnecessary because you never user/reference them. This is a problem in other parts of our code too but we should aim for minimal code in the future.

1. you don't need to assign objects to "self" if you never reference them, e.g. self.order, self.hardware etc. you can simply call the objects.create() function and the data will be created.
2. you don't need to create so many objects. you never query the hardware or order items tables so you don't need to create multiple of them, always try to do the minimum amount when you create your setup data. So here you only need 1 hardware, 1 order_item and 1 incident
3. your data needs to add up. you first create an order with:
       2 items with hardware id 1
       3 items with hardware id 2
But you only create 1 order item with hardware id 1 and 1 order item with hardware id 2, meanwhile you should create as many order items as there are in the order you specify. Just something to keep in mind. Going off of point 2 though, you should change your order request to only request 1 item for minimal db setup.